Cargo.toml

```diff
@@ -19,8 +19,11 @@ members = [
     "tests/runtimes/simple-keyvalue",
     "tests/runtimes/simple-consensus",
     "tests/runtimes/simple-evm",
+
+    # OAS contracts specs.
+    "runtime-sdk/modules/contracts/specs/oas20",
```

Comment:
It would probably make more sense to put these under `contract-sdk/contracts`?

Reply:
Hm yeah i actually had it there earlier, then I figured that the code actually cannot only depend on the `contract-sdk` crates, but also the `runtime-sdk/modules/contracts` due to the Id->Address derivation for contracts (which is defined in `modules/contracts` and also depends on MODULE_NAME).

However even with that dependency i think it does make more sense that it lives in `contract-sdk/contracts` for now. I'll move it back.

Reply:
Hm the contract part should not depend on the runtime SDK as then it will not be usable from contracts. If needed, we should copy some (minimal) functionality into the `oasis-contract-sdk-types` crate which is shared between the runtime SDK module and the contract SDK. This is what has been done for things like the `Address` type.

Reply:
Oh. For context i needed for this:
https://github.com/oasisprotocol/oasis-sdk/blob/bf3d54dc7a3ec1e76994353be566cdf15a2489fd/runtime-sdk/modules/contracts/src/types.rs#L85-L87

so I could get a contract address from its instance id. One option would be moving this id->address mapping to the `oasis-contract-sdk-types` (and make it not rely on module name?).  Or having access to the id->address (and otherway) mapping available somewhere in the `context` or `env`.